### PR TITLE
[Metricbeat][MySQL] Performance metricset does not working with MySQL 5.7

### LINKED ISSUE #25069: Performance metricsets of Metricbeat MySQL module does not working with MySQL 5.7
- Version: 7.10.x
- Steps to Reproduce: 
  - Install MySQL 5.7
  - Install Metricbeat with enabling MySQL module
  - Enable performance metricsets for the installed MySQL

Metricbeat - MySQL - performance metricsets fails with the following errors when enabling it for MySQL 5.7.
```
ERROR [mysql.performance] query/query.go:92 error doing query {events_statements SELECT digest_text, count_star, avg_timer_wait, max_timer_wait, last_seen, quantile_95
FROM performance_schema.events_statements_summary_by_digest
ORDER BY avg_timer_wait DESC
LIMIT 10
table %!s(bool=true)}%!(EXTRA *mysql.MySQLError=Error 1054: Unknown column 'quantile_95' in 'field list')
```

For the `quantile_95` field, I can find it on [8.0 Manual](https://dev.mysql.com/doc/refman/8.0/en/statement-summary-tables.html) but not on [5.7 manual](https://dev.mysql.com/doc/refman/5.7/en/statement-summary-tables.html). So, it seems this metricsets is not supported MySQL 5.7 but our document says MySQL module is expected to work with all versions >=5.7.0.
> The mysql MetricSets were tested with MySQL and Percona 5.7 and 8.0 and are expected to work with all versions >= 5.7.0. It is also tested with MariaDB 10.2, 10.3 and 10.4.

If the `quantile_95` column is required, we need to clarify this restriction in the documentation.

### PR DESCRIPTION
**Pull request by kush-elastic**
## Description

The following query is failing to run on MySQL version 5.7 because the quantile_95 is added as part of a new metric in MySQL version `8.0.1`.
```
SELECT digest_text, count_star, avg_timer_wait, max_timer_wait, last_seen, quantile_95
    FROM performance_schema.events_statements_summary_by_digest
    ORDER BY avg_timer_wait DESC
    LIMIT 10
```

Updated Query to work with both version 5.7 and 8.0 with MySQL executable support. with version check, query parameters will be skipped if version 5.7 and 8.0.0 (which both do not have support for `quantile_95`).
```
SELECT digest_text, count_star, avg_timer_wait, max_timer_wait, last_seen /*!80001 ,quantile_95 */ 
    FROM performance_schema.events_statements_summary_by_digest 
    ORDER BY avg_timer_wait DESC 
    LIMIT 10;
```

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding changes to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [x] Check the query with the different versions from MariaDB and Percona

## Related issues
- Closes #25069

## Logs
```
ERROR [mysql.performance] query/query.go:92 error doing query {events_statements SELECT digest_text, count_star, avg_timer_wait, max_timer_wait, last_seen, quantile_95
FROM performance_schema.events_statements_summary_by_digest
ORDER BY avg_timer_wait DESC
LIMIT 10
table %!s(bool=true)}%!(EXTRA *mysql.MySQLError=Error 1054: Unknown column 'quantile_95' in 'field list')
```

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -108,6 +108,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix Azure Monitor 429 error by causing metricbeat to retry the request again. {pull}38294[38294]
 - Fix fields not being parsed correctly in postgresql/database {issue}25301[25301] {pull}37720[37720]
 - rabbitmq/queue - Change the mapping type of `rabbitmq.queue.consumers.utilisation.pct` to `scaled_float` from `long` because the values fall within the range of `[0.0, 1.0]`. Previously, conversion to integer resulted in reporting either `0` or `1`.
+- Fix for MySQL/Performance - Query failure in case of MySQL version 5.7 for `quantile_95`. {pull}38710[38710]
```
Comment (agithomas) [inline CHANGELOG.next.asciidoc:111]: Just 5.7 or all versions prior to 8.0.0 ?

Comment (shmsr) [reply CHANGELOG.next.asciidoc:111]: All versions prior to 8.0 are affected. We should update this changelog update.

Comment (shmsr) [reply CHANGELOG.next.asciidoc:111]: To be very precise it was introduced in 8.0.1

See: https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-1.html#:~:text=In%20addition%2C%20the%20events_statements_summary_by_digest%20table%20now%20has%20columns%20that%20provide%20percentile%20information%20about%20statement%20latencies.%20For%20more%20information%2C%20see%20Statement%20Summary%20Tables.

Even Kush pointed it out here: https://github.com/elastic/beats/issues/25069#issuecomment-2029319307 (8.0.1)

Comment (kush-elastic) [reply CHANGELOG.next.asciidoc:111]: I will update the Change log, Thanks for bringing this up.

File: CHANGELOG.next.asciidoc [hunk 2]
```diff
@@ -111,6 +111,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix Azure Monitor 429 error by causing metricbeat to retry the request again. {pull}38294[38294]
 - Fix fields not being parsed correctly in postgresql/database {issue}25301[25301] {pull}37720[37720]
 - rabbitmq/queue - Change the mapping type of `rabbitmq.queue.consumers.utilisation.pct` to `scaled_float` from `long` because the values fall within the range of `[0.0, 1.0]`. Previously, conversion to integer resulted in reporting either `0` or `1`.
+- Fix for MySQL/Performance - Query failure for below version 8.0.1 of MySQL, for performance metric `quantile_95`. {pull}38710[38710]
```
Comment (shmsr) [inline CHANGELOG.next.asciidoc:114]:
```suggestion
- Fix for MySQL/Performance - Query failure for MySQL versions below v8.0.1, for performance metric `quantile_95`. {pull}38710[38710]
```

**Review by shmsr — APPROVED** [review]

**Pull request merged**